delta/service/src/test/resources/akka-test.conf

```diff
@@ -0,0 +1,67 @@
+akka {
+  actor {
+    provider = "akka.cluster.ClusterActorRefProvider"
+    warn-about-java-serializer-usage = off
+
+    serializers {
+      kryo = "io.altoo.akka.serialization.kryo.KryoSerializer"
+    }
+
+    serialization-bindings {
+      "ch.epfl.bluebrain.nexus.delta.sdk.model.permissions.PermissionsCommand"   = "kryo"
```

Comment:
Is there some better way for kryo serialization instead of this config file? Annotations, marker trait?

Reply:
I think it's a good idea to keep these explicit. One needs to know how things are serialised, especially events.